src/bun.js/test/jest.zig

```diff
@@ -1375,6 +1367,12 @@ pub const TestRunnerTask = struct {
         jsc_vm.onUnhandledRejectionCtx = this;
         jsc_vm.onUnhandledRejection = onUnhandledRejection;
 
+        if (this.describe.runCallback(globalThis, .beforeAll)) |err| {
```

Comment:
Instead we can trigger the `beforeAll` blocks right before the `beforeEach` blocks